spec/rubocop/cop/rails/unique_validation_without_index_spec.rb

```diff
@@ -64,14 +64,23 @@ class User
         end
       RUBY
 
-      it 'registers an offense' do
+      it 'registers an offense when the `uniqueness: true` option is used alone' do
         expect_offense(<<~RUBY)
           class User
             validates :account, uniqueness: true
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Uniqueness validation should have a unique index on the database column.
           end
         RUBY
       end
+
+      it 'registers an offense when the `presence: true` option is used alone' do
+        expect_offense(<<~RUBY)
+          class User
+            validates :account, presence: true
+            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Uniqueness validation should have a unique index on the database column.
```

Comment:
Should this really register an offense in this case? Doesn't make sense in my opinion.

Reply:
I also think this should not raise an error.

Think about this example:
```
class Post
  belongs_to :user

	validates :user, presence: true
end

class User
  has_many :posts
end
```

The column `user_id` in `posts` can NEVER be unique because a user can have many posts.

Reply:
Thank you for the comments. This is a bug and I'm working now.

Reply:
I've updated this PR.